alpha/veneer/basic/README.md

```diff
@@ -0,0 +1,117 @@
+### Basic Veneer
+To simplify operation with File-Based Catalogs (FBCs), we can first offer the option to eliminate information which can be retrieved from existing registries when we generate the FBC. The `basic veneer` is an input schema which is effectively a subset of an FBC with support to do just this. Users of the `basic veneer` can provide all existing elements of an FBC, but short-hand the `olm-bundle` objects, requiring only the bundle image reference and greatly reducing the size/complexity of the resulting document.
+
+
+
+In a very simple example, we define a simple `olm.package` and an `olm.channel` composed of two `olm.bundle` objects which have an image name attribute but no other attributes/properties.
```

Comment:
```suggestion
In a very simple example, we define an `olm.package` and an `olm.channel` composed of two `olm.bundle` objects that have an image name attribute but no other attributes/properties.
```

Reply:
I de-simpled the excessively simple-adorned simple sentence, simplifying to simpler structure with a desired dearth of 'simple's.  
;)